src/grafana_dashboards/temporal-monitoring.json.tmpl

```diff
@@ -0,0 +1,2942 @@
+{
```

Comment:
Do you have any screenshots of the dashboard output? It's hard to review purely in this format. Or is it the same as what we have for the server?

Reply:
Nope this is SDK-general dsahboard different from the server dashboard. Attaching some screenshots to PR definition.